lib/generic/modules/arthritis.json

```diff
@@ -0,0 +1,737 @@
+{
+  "name": "Arthritis",
+  "remarks": [
+    "Arthritis falls into several broad categories: ",
+    "1. Osteoarthritis       - a breakdown of cartilage in the joints ",
+    "2. Rheumatoid Arhtritis - joints and other organs are attacked by the immune system ",
+    "3. Fibromyalgia         - central nervous system, widespread chronic pain ",
+    "4. Gout                 - painful inflammation of the joints caused by a buildup of uric acid ",
+    "5. Lupus                - widespread inflammatory autoimmune disease ",
+
+    "Information on treatment, prevalence, and incidence of specific types from the CDC: ",
+    "http://www.cdc.gov/arthritis/basics/types.html",
+
+    "Overall prevalence: ",
+    "http://www.cdc.gov/arthritis/data_statistics/national-statistics.html"
+  ],
+  "states": {
+
+    "Initial": {
+      "type": "Initial",
+      "remarks": [
+        "======================================================================",
+        " INCIDENCE                                                            ",
+        "======================================================================",
+
+        "While there are some cases where race is a factor in prevalence (lupus and gout are ",
+        "more likely in blacks and natives) in general the differences between males and females ",
+        "are much greater. For simplicity I'm omitting the differences in race for the time being.",
+
+        " | Arthritis Type | Prevalence | Ratio F:M | ",
+        " ------------------------------------------- ",
+        " | Osteoarthritis |   0.34000  |    2:1    | ",
+        " | Fibromyalgia   |   0.01670  |    7:1    | ",
+        " | Gout           |   0.03900  |    1:3    | ",
+        " | Rheumatoid     |   0.01000  |    1:1    | ",
+        " | Lupus          |   0.00054  |    6:1    | ",
+        " ------------------------------------------- "
+      ],
+      "direct_transition": "Delay_Until_Arthritis"
+    },
+
+    "Delay_Until_Arthritis": {
```

Comment:
In this model people can only have one of these 5 conditions. Is that true in reality? I would assume there's some level of comorbidity, in particular with gout which is heavily dependent on diet, or with osteoarthritis in multiple locations.